nucypher/cli/characters/felix.py

```diff
@@ -85,42 +104,64 @@ def felix(click_config,
 
         return  # <-- do not remove (conditional flow control)
 
-    #
-    # Authenticated Configurations
-    #
-
     # Domains -> bytes | or default
-    domains = [bytes(network, encoding='utf-8')] if network else None
+    domains = [network] if network else None
```

Comment:
What does this mean?
Why is network coerced into an array?
Can we have a comment?

Reply:
what if network _is_ an array?  Can it be?
How would I specify that?

Reply:
At the moment, the current API support one one domain passed as a CLI flag `--network`, However, under the hood we actually support `Learner`/`Teacher`s on multiple domains in a `set`.  This is entry-point code for the domain.  Perhaps it can be cast to a `set` here instead, or better yet, we can integrate variadic arguments (see <https://click.palletsprojects.com/en/5.x/arguments/#variadic-arguments>).

Reply:
Follow-up Issue https://github.com/nucypher/nucypher/issues/1042